apps/end2end/clock/regions/stopwatch.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+var PageRegion = require(
+    '../../../../shared/test/integration/helpers/page_region');
+var MarionetteSearchTypes = require(
+    '../../../../shared/test/integration/helpers/marionette_search_types');
+
+function Stopwatch(client) {
+    PageRegion.call(this, client,
+        Stopwatch.prototype.selectors._stopwatchViewLocator);
+    var viewLocator = this.findElement(
```

Comment:
This shouldn't be called viewLocator, since this isn't a locator, but an element.